BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -220,6 +211,29 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
     public func didPressEnterManually() {
         self.resultsDelegate?.giniCaptureDidEnterManually()
     }
+
+    public func startSDK(withDocuments documents: [GiniCaptureDocument]?, animated: Bool = false) -> UIViewController {
+        configurationService?.fetchConfigurations(completion: { result in
+            switch result {
+            case .success(let configuration):
+                self.initializeAnalytics(with: configuration)
+            case .failure(let error): 
+                break
+                /* There will be no retries if the endpoint fails.
+                We will not implement any caching mechanism on our side if the request is too slow.
+                In case of a failure, the UJ analytics will remain disabled for that session. */
+            }
```

Comment:
[clarification]: This will be changed with this ticket https://ginis.atlassian.net/browse/PP-539 or at least we can remove the comment :D